changelog/1.19.0.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "1.19.0"
+description: "Released on 05/19/2021"
+---
+
+### Breaking changes ❗
+
+There are no breaking changes in 1.19.0.
+
+### Features ✨
+
+- web: Resource quotas have been introduced to Organizations. Resource Quotas define
+  maximum resource utilization applied to each member within the Organization they're
+  applied to. Modify them **Manage > Organizations > Edit Organization**.
+- web: Add site and organization workspace as code templates
+- web: An administrative control over Dev URL access levels was added.
+- web: By default, the CVM option for creating custom environments is now checked
+- web: Improved display and support for errors returned by an IdP during sign in
+- infra: Use stdio for envassets
+
+### Bug fixes 🐛
+
+- web: Fixed an issue whereby in the **Edit Workspace** dialog, resource selectors
+  displayed as disabled but still allowed modification
+- web: Updated validation logic so that workspaces with invalid names can no longer
+  be created
+- infra: Fixed an issue that caused workspace autostart to sometimes halt or fail
+
+### Security updates 🔐
+
+- web: Upgrade version of NextJS to 10.2.0
```

Comment:
perhaps we should do a scan of 1.18.1 and the latest version and list out all the fixed CVEs?

Reply:
I'm good w that - will need help of @mterhar